apps/api-server/src/adapter/openstad/service.js

```diff
@@ -185,6 +185,34 @@ service.deleteUser = async function({ authConfig, userData = {} }) {
 
 }
 
+service.fetchClient = async function({ authConfig, project }) {
+
+  let clientId = authConfig.clientId;
+  if (!clientId) {
+    throw new Error('OpenStad.service.updateClient: clientId not found')
+  }
+
+  try {
+
+    let url = `${authConfig.serverUrlInternal}/api/admin/client/${clientId}`;
+    let response = await fetch(url, {
+	    headers: {
+        Authorization: `Basic ${new Buffer(`${authConfig.clientId}:${authConfig.clientSecret}`).toString('base64')}`,
+      },
+    })
+    if (!response.ok) {
+      console.log(response);
```

Comment:
Deze mag nog weg

Reply:
Die zijn al verdwenen met je vorige commentaar, maar dat zit hier nog niet in :)